src/renderer/src/components/ColorSelector.tsx

```diff
@@ -0,0 +1,61 @@
+import { useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { MdAdd } from 'react-icons/md'
+import { TAccountColorKey } from '@renderer/@types/blockchain'
+import { IAccountState } from '@renderer/@types/store'
+import { ReactComponent as CheckWhite } from '@renderer/assets/images/check-white.svg'
+import { StyleHelper } from '@renderer/helpers/StyleHelper'
+
+type TProps = {
+  account: IAccountState
+  label: string
+  setNewColor: (color: string) => void
+}
+
+export const ColorSelector = ({ account, label, setNewColor }: TProps) => {
+  const { t } = useTranslation('components', { keyPrefix: 'colorSelector' })
+  const accountColorToSelect: string[] = ['green', 'blue', 'purple', 'magenta', 'orange', 'yellow', 'gray']
+  const gradientByAccountColor: Record<TAccountColorKey, string> = {
+    blue: 'from-[#242171] to-[#919BED]',
+    green: 'from-[#164C55] to-[#63D5D0]',
+    gray: 'from-[#324C5F] to-[#9ABED3]',
+    magenta: 'from-[#4C2F71] to-[#BB9EE4]',
+    yellow: 'from-[#BDA133] to-[#FEDD5B]',
+    purple: 'from-[#2F008E] to-[#7C4BFE]',
+    orange: 'from-[#B44D00] to-[#FE872F]',
+  }
```

Comment:
Put it outside the component
```suggestion
  const accountColorToSelect: string[] = ['green', 'blue', 'purple', 'magenta', 'orange', 'yellow', 'gray']
  const gradientByAccountColor: Record<TAccountColorKey, string> = {
    blue: 'from-[#242171] to-[#919BED]',
    green: 'from-[#164C55] to-[#63D5D0]',
    gray: 'from-[#324C5F] to-[#9ABED3]',
    magenta: 'from-[#4C2F71] to-[#BB9EE4]',
    yellow: 'from-[#BDA133] to-[#FEDD5B]',
    purple: 'from-[#2F008E] to-[#7C4BFE]',
    orange: 'from-[#B44D00] to-[#FE872F]',
  }
  
  export const ColorSelector = ({ account, label, setNewColor }: TProps) {
  ...
  }
```